nautobot/core/views/mixins.py

```diff
@@ -252,7 +252,7 @@ def get_required_permission(self):
         """
         queryset = self.get_queryset()
         try:
-            actions = [PERMISSIONS_ACTION_MAP[self.action]]
+            actions = [PERMISSIONS_ACTION_MAP.get(self.action, "view")]
```

Comment:
This is a different approach than what's being done in #4512 to address #4529, so we should consider that carefully. Also I'm not sure `view` is the appropriate default permission to use here.